### PR TITLE
Feature/jude/user-upload-avatar

### PR DESCRIPTION
**Pull request by judeokafor**
A protected route that allows users upload profile picture

### CODE REVIEW

File: functions/.eslintrc.json [hunk 1]
```diff
@@ -82,7 +82,7 @@
 		"max-len": ["error", {
 			"code": 100
 		}],
-		"max-lines-per-function": ["error", 25],
+		"max-lines-per-function": ["error", 30],
```
Comment (Way2nnadi) [inline functions/.eslintrc.json:85]: Why did you change this rule?

File: functions/src/utils/firebase.js [hunk 2]
```diff
@@ -13,6 +13,7 @@ const config = {
 admin.initializeApp({
     credential: admin.credential.cert(serviceAccount),
     databaseURL: databaseURL,
+    storageBucket: storageBucket,
```
Comment (Way2nnadi) [inline functions/src/utils/firebase.js:16]: given that the key and value are the same you can leave off the value
```
...
databaseURL,
storageBucket,
...
```

File: functions/.eslintrc.json (hunk 1 above)
Comment (judeokafor) [reply functions/.eslintrc.json:85]: I was checking out something and forgot revert back. I will revert back

File: functions/src/utils/firebase.js (hunk 2 above)
Comment (judeokafor) [reply functions/src/utils/firebase.js:16]: Okay. Wil do that

**Comment by Simpcyclassy** [comment]
Fix the changes, and every other thing looks good to me.
